src/tests/mock/service.ts

```diff
@@ -427,7 +427,139 @@ const server = setupServer(
         siafi: "9701",
       })
     );
-  })
+  }),
+  rest.get("https://localhost:7083/api/solicitacaoAcao/escolas", (req, res, ctx) => {
```

Comment:
Trocar o localhost pela rota do deploy

Reply:
O PR do back ainda n foi aceito

Reply:
Mas aqui o no front tem que estar a rota certa, o localhost não funciona aqui. Quando o pr do back for aceito a rota vai estar funcionando